examples/plots/spc_convective_outlook.py

```diff
@@ -0,0 +1,50 @@
+#  Copyright (c) 2021 MetPy Developers.
+#  Distributed under the terms of the BSD 3-Clause License.
+#  SPDX-License-Identifier: BSD-3-Clause
+"""
+NOAA SPC Convective Outlook
+===========================
+
+Demonstrate the use of geoJSON and shapefile data with PlotGeometry in MetPy's simplified
+plotting interface. This example walks through plotting the Day 1 Convective Outlook from NOAA
+Storm Prediction Center. The geoJSON file was retrieved from the `Storm Prediction Center's
+archives <https://www.spc.noaa.gov/archive/>`_.
+"""
+
+import geopandas
+
+from metpy.cbook import get_test_data
+from metpy.plots import MapPanel, PanelContainer, PlotGeometry
+
+###########################
+# Read in the geoJSON file containing the convective outlook.
+day1_outlook = geopandas.read_file(get_test_data('spc_day1otlk_20210317_1200_lyr.geojson'))
+
+###########################
+# Preview the data.
+day1_outlook
```

Comment:
I prefer Pandas dataframes in notebooks displayed this syntax `day1_outlook` rather than `print(day1_outlook)`. However, I understand that this isn't strictly a notebook example and that both .ipynb and .py versions of this example will be available to users. I don't mind using `print(day1_outlook)` here if you all would prefer that.

Reply:
Eh, the nicer display in docs in good to have--certainly not worth sacrificing to please a tool. I marked it to ignore and told Codacy to re-scan.